src/protocols/kubernetes/settings.c

```diff
@@ -25,6 +25,14 @@
 
 /* Client plugin arguments */
 const char* GUAC_KUBERNETES_CLIENT_ARGS[] = {
+    /**
+     * This first argument defines the protocol version in use so that the
+     * client knows how to handle talking to different versions of guacd.
+     * If this is omitted the client may choose not to enable certain
+     * features.
+     */
+    "VERSION_1_1_0",
```

Comment:
Any particular reason behind the formatting of the version string?

Reply:
It matches the `enum` on the Java side of things that I implemented, so it makes it easy to use `valueOf` in Java to pull the matching value.  I'm not particularly tied to that - originally I had something like `version:1.1.0` and could go back to that, or whatever else we decide we want to do for that argument.

Reply:
`VERSION_1_1_0` seems fine to me, and would indeed be convenient for parsing on the Java side.

This would remain `VERSION_1_1_0` even after guacamole-server is at 1.2.0, 1.3.999, etc., being a reference to the protocol variation introduced by the release having that same version number?

Reply:
> This would remain VERSION_1_1_0 even after guacamole-server is at 1.2.0, 1.3.999, etc., being a reference to the protocol variation introduced by the release having that same version number?

I don't have a strong preference either way, but that would be fine with me.  Unless you think it adds confusion to have the protocol version follow the product version?  I think it would be okay, as it would establish the minimum software version for each of the protocol features, which seems reasonably to me.